frontend/src/v5/store/federations/federations.types.ts

```diff
@@ -75,6 +75,8 @@ export type FederationSettings = Omit<FederationBackendSettings, 'surveyPoints'>
 
 export type MinimumFederation = Pick<IFederation, '_id' | 'name' | 'role' | 'isFavourite'>;
 
+export type FederationRevision = Pick<FederationBackendSettings, '_id' | 'timestamp' >;
```

Comment:
Should this also contain "tag" and "author" as specified [here](https://github.com/3drepo/3drepo.io/issues/3312)?

Reply:
I think you are right!